lib/asciidoctor/converter/semantic_html5.rb

```diff
@@ -67,6 +67,150 @@ def convert_paragraph node
     end
   end
 
+  def convert_thematic_break node
+    "<hr />"
+  end
+
+  def convert_admonition node
+    attributes = common_html_attributes node.id, node.role, %(admonition #{node.attr 'name'})
+    ret = []
+    ret << %(<aside#{attributes}>)
+    ret << %(<em class="label">#{node.attr "textlabel"}</em>)
+    ret << %(<strong class="title">#{node.title}</strong>) if node.title?
+    ret << node.content
+    ret << "</aside>"
+    ret.join "\n"
+  end
+
+  def convert_ulist node
+    attributes = common_html_attributes node.id, node.role
+    ret = []
+    ret << if node.title?
+      %(<ul#{attributes}>
+<strong class="title">#{node.title}</strong>)
+    else
+      %(<ul#{attributes}>)
+    end
+    node.items.each do |item|
+    sub_content = if item.blocks?
+      %(
+#{item.content}
+)
+    else
+      ""
+    end
+    ret << %(<li>#{item.text}#{sub_content}</li>)
```

Comment:
I think we need a wrapping element if the content is compound.

Reply:
I believe that `#{sub_content}` has a wrapper already. See [nested-unordered-list.html](https://github.com/asciidoctor/asciidoctor/pull/4235/files#diff-f75090d4025411eb5ce481ddbfa1bbe7889446c32418b2082b86280e6899a1f6). Maybe that isn't the case with all potential sub_contents?

Reply:
I believe this also applys to `convert_olist` (L134) and `convert_dlist` (L167), so I'll hold off on moving them too.

Reply:
let's continue the discussion on https://github.com/asciidoctor/asciidoctor/pull/4321